railties/lib/rails/generators/rails/app/templates/config/application.rb.tt

```diff
@@ -50,5 +50,8 @@ module <%= app_const_base %>
     # Don't generate system test files.
     config.generators.system_tests = nil
 <%- end -%>
+
+    # Strip trailing newlines from rendered output.
+    # config.action_view.strip_trailing_newlines = false
```

Comment:
I don't think we should generate this comment for every new application, given that similar placeholders were removed in https://github.com/rails/rails/commit/28dcadc0140dfdebe87d5e691fd709c0a9ae0bae because they weren't changed often to warrant inclusion.

Reply:
Not sure I agree — I think it's too early to confirm this.